src/component/Header/index.jsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import Nav from 'react-bootstrap/Nav'
+import Navbar from 'react-bootstrap/Navbar'
+import Image from 'react-bootstrap/Image'
+import styles from './Header.module.css'
+import PomoLogo from '../../Logo/PomoLogo.png'
+import SettingLogo from '../../Logo/SettingLogo.png'
+import LoginLogo from '../../Logo/LoginLogo.png'
 
-const Header = () => (
-  <div>
-    <h1>hi</h1>
-  </div>
-);
+const Header = () => {
+  return (
+    <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+      <Image className={styles.pomo_logo} src={PomoLogo} rounded />
+      <Navbar.Brand href="#home">TheSol-pomodoro</Navbar.Brand>
+      <Navbar.Toggle />
+      <Navbar.Collapse id="responsive-navbar-nav">
```

Comment:
This `id` is not being used and should be removed

```suggestion
      <Navbar.Collapse>
```